fdbclient/ReadYourWrites.actor.cpp

```diff
@@ -1189,6 +1189,43 @@ ACTOR Future<Standalone<RangeResultRef>> getWorkerInterfaces (Reference<ClusterC
 	}
 }
 
+ACTOR Future<Standalone<RangeResultRef>> getConflictingKeys(
```

Comment:
This should not be an actor. Instead it should assert that it never needs to wait on anything (see below)

Reply:
The code is already refactored as we discussed. Thanks.

Reply:
The code is refactored as we discussed.